src/test/java/mops/presentation/GroupControllerTest.java

```diff
@@ -65,13 +64,26 @@
     void setUp() {
         account = new Account("studi", "bla@bla.de", "studentin");
         given(fileService.getAllFilesOfGroup(account, 1)).willReturn(List.of());
+        given(groupService.getGroupURL(account, 1)).willReturn(new GroupURLWrapper(1L));
         mvc = MockMvcBuilders
                 .webAppContextSetup(context)
                 .alwaysDo(print())
                 .apply(springSecurity())
                 .build();
     }
 
+    /**
+     * Tests the API for getting the group url.
+     */
+    @Test
+    public void getGroupURL() throws Exception {
+        setupSecurityContextMock(account);
+        mvc.perform(get("/material1/group/1/url"))
+                .andExpect(status().is2xxSuccessful())
+                .andExpect(jsonPath("$.url").value("/dir/1"))
+                .andExpect(jsonPath("$.group_id").value("1"));
```

Comment:
```suggestion
                .andExpect(jsonPath("$.group_id").value(1));
```

Reply:
```suggestion
                .andExpect(jsonPath("$.group_id").value(1L));
```
we could actually use `1L`